04_data-viz.qmd

```diff
@@ -555,45 +653,67 @@ storms %>%
   geom_line(mapping = aes(x = year, y = n))
 ```
 
-**`r kableExtra::text_spec("Step 2:", color = "#55b748")`** Add `geom_point()` after `geom_line()` with the same aesthetic mappings.
+2. Add `geom_point()` after `geom_line()` with the same aesthetic mappings.
+:::
 
-\vspace{0.25in}
+### Layers (bonus!): 
 
 :::{.callout-note}
 **Layers** allow for multiple geometric objects to be plotted in the same data visualization.
 :::
 
-\vspace{0.25in}
 
-### `r kableExtra::text_spec("Exercise 10", color = "#1696d2")`
 
-**`r kableExtra::text_spec("Step 1:", color = "#55b748")`** Add the following exercise to you script. Run it!
+::: callout
+
+#### [`r paste("Exercise", exercise_number)`]{style="color:#1696d2;"}
+
+```{r}
+#| echo: false
+
+exercise_number <- exercise_number + 1
+```
+
+1. Add the following exercise to you script. Run it!
 
 ```
 ggplot(data = storms, mapping = aes(x = pressure, y = wind)) + 
   geom_point() +
   geom_smooth()
 ```
+:::
 
-\vspace{0.25in}
+### Inheritances (bonus!):
 
 :::{.callout-note}
 **Inheritances** pass aesthetic mappings from `ggplot()` to later `geom_*()` functions.
 :::
 
-\vspace{0.25in}
 
-\textit{Notice how the aesthetic mappings are passed to} `ggplot()` \textit{in example 10. This is useful when using layers!}
+Notice how the aesthetic mappings are passed to `ggplot()` in example 10. This is useful when using layers!
 
-### `r kableExtra::text_spec("Exercise 11", color = "#1696d2")`
 
-**`r kableExtra::text_spec("Step 1:", color = "#55b748")`** Pick your favorite plot from exercises 1 through 10 and duplicate the code.
+::: callout
 
-**`r kableExtra::text_spec("Step 2:", color = "#55b748")`** Add `ggsave(filename = "favorite-plot.png")` on a new line without `+` and then save the file. Look at the saved file. 
+#### [`r paste("Exercise", exercise_number)`]{style="color:#1696d2;"}
+
+```{r}
+#| echo: false
+
+exercise_number <- exercise_number + 1
+```
+
+1. Pick your favorite plot from exercises 1 through 10 and duplicate the code.
+
+2. Add `ggsave(filename = "favorite-plot.png")` on a new line without `+` and then save the file. Look at the saved file. 
+
+3. Add `width = 6` and `height = 4` to `ggsave()`. Run the code and then look at the saved file. 
+:::
 
-**`r kableExtra::text_spec("Step 3:", color = "#55b748")`** Add `width = 6` and `height = 4` to `ggsave()`. Run the code and then look at the saved file. 
+## Summary
```

Comment:
Seems like we should add summary sections everywhere or no where.

Reply:
That is fair! I think it could be helpful. I will add another issue and assign it to me.